Example/Pods/Local Podspecs/Heartland-iOS-SDK.podspec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Heartland-iOS-SDK",
+  "version": "1.0.6",
+  "summary": "Secure Tokenized Payments by Heartland Payment Systems.",
+  "description": "Secure Tokenized Payments for iOS by Heartland Payment Systems.",
+  "homepage": "https://github.com/hps/heartland-ios",
+  "license": "EULA",
+  "authors": {
+    "Shaunti Fondrisi": "shaunti.fondrisi@e-hps.com"
```

Comment:
Can we update this to point to the team instead? `Heartland Developer Portal <EntApp_DevPortal@e-hps.com`